ppa/archive/models.py

```diff
@@ -121,6 +122,76 @@ def stats():
         return stats
 
 
+class ProtectedFlags(Flags):
```

Comment:
These flags are very much specific to `DigitizedWork`, so I think the name should indicate that more clearly. I'm not sure how to do that without making it super long, though. `ProtectedWorkFields` maybe? Or are you using flags here to distinguish from the model field? `ProtectedWorkFieldFlags` ?

Reply:
That was the use of `Flags` (distinguishing from the model field). I like the two proposed options.

Reply:
Actually, I like `ProtectedWorkFieldFlags` and `ProtectedWorkField` because it's a `models.Field` subclass.